### PR TITLE
Announcements folder; create triage-wg-welcome

### PR DESCRIPTION
**Pull request by BatmanAoD**
First draft of triage wg announcement

@Dylan-DPC

### CODE REVIEW

File: announcements/triage-wg-welcome.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+The goal is to prevent issues and pull requests from being ignored by the
+Rust team or forgotten by the contributors who submit them, and to ensure
+that everyone involved in the process knows what the next step is in the process
+toward resolving an issue or merging a pull request. In a large project
+like Rust, keeping up with community contributions is one of the main
+difficulties when scaling up development efforts, so the triage WG's efforts
+will alleviate a key pain point in Rust's growth.
+
+Triaging does not require reading or writing code. If you like the Rust project
+and would like to help it advance but do not feel comfortable submitting
```
Comment (Centril) [inline announcements/triage-wg-welcome.md]: Well if you are comfortable with submitting PRs you can and should still join... 😂

Comment (BatmanAoD) [reply announcements/triage-wg-welcome.md]: Hm, I didn't mean for that to sound exclusionary, but you're right, it can be taken that way. I've removed that paragraph and added the "not require...code" bit to its own sentence in the second paragraph. Thanks!

File: announcements/triage-wg-welcome.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+commitment.  It also does not require reading or writing code.  The goal of
+triage is to prevent issues and pull requests from being ignored by the Rust
+team or forgotten by the contributors who submit them, and to ensure that
+everyone involved in the process knows what the next step is in the process
+toward resolving an issue or merging a pull request. In a large project like
+Rust, keeping up with community contributions is one of the main difficulties
+when scaling up development efforts, so the triage WG's efforts will alleviate
+a key pain point in Rust's growth.
+
+If you're interested in becomoing a triage WG member, please leave a message in
```
Comment (Dylan-DPC-zz) [inline announcements/triage-wg-welcome.md:16]: Typo in becoming

File: announcements/triage-wg-welcome.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+The Release team is pleased to announce the **triage working group**, which
+will assume responsibility for triaging issues and pull requests in the
+[rust-lang](https://github.com/rust-lang) repositories as well as for
+diagnosing failures in Crater runs.
+
+Triaging is a high-impact activity that requires a relatively small time
```
Comment (Dylan-DPC-zz) [inline announcements/triage-wg-welcome.md:6]: I'd rephrase that as "requires a relatively less time"

**Review by Dylan-DPC-zz — COMMENTED** [review]
Except a few tidbits, looks good to me. You can correct them or else not an issue as I'll fix them when creating the post

File: announcements/triage-wg-welcome.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+The Release team is pleased to announce the **triage working group**, which
+will assume responsibility for triaging issues and pull requests in the
```
Comment (Mark-Simulacrum) [inline announcements/triage-wg-welcome.md:2]: I would note here that this is taking responsibility from the release team just to show who is currently in charge of this

File: announcements/triage-wg-welcome.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+[rust-lang](https://github.com/rust-lang) repositories as well as for
+diagnosing failures in Crater runs.
+
+Triaging is a high-impact activity that requires a relatively small time
+commitment.  It also does not require reading or writing code.  The goal of
+triage is to prevent issues and pull requests from being ignored by the Rust
```
Comment (Mark-Simulacrum) [inline announcements/triage-wg-welcome.md:8]:
```suggestion
triage is to prevent issues and pull requests from being missed by the Rust
```

**Review by Mark-Simulacrum — APPROVED** [review]
Looks good overall. I would suggest posting this on users and internals whenever you're ready. We'll want to be sure to have a good process for people to follow before we do though -- maybe a signup sheet and an update to the directions on the forge incorporating triagebot (rustbot) commands for users without rust-lang/rust permissions.

If there's some updates to triagebot that are necessary or would make that easier I can spend some time adding those tomorrow.
